infra/app/storage.bicep

```diff
@@ -0,0 +1,64 @@
+param storageAccountName string = 'Enabled'
+param location string = ''
+param blobContainerName string = 'Enabled'
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2021-09-01' = {
+  name: storageAccountName
+  location: location
+  kind: 'StorageV2'
+  sku: {
+    name: 'Standard_GRS'
+  }
+}
+
+resource storageAccountNameDefaultBlobContainer 'Microsoft.Storage/storageAccounts/blobServices/containers@2021-08-01' = {
```

Comment:
Move all of these as child resources of the storageAccount.

Remove all the dependsOn